File: python/1038.py
codigo, quantidade = map(float, input().split(' '))

if codigo == 1:
    x = quantidade * 4.00
    print("Total: R$ {0:.2f}".format(x))

elif codigo == 2:
    x = quantidade * 4.50
    print("Total: R$ {0:.2f}".format(x))

elif codigo == 3:
    x = quantidade * 5.00
    print("Total: R$ {0:.2f}".format(x))

elif codigo == 4:
    x = quantidade * 2.00
    print("Total: R$ {0:.2f}".format(x))

elif codigo == 5:
    x = quantidade * 1.50
    print("Total: R$ {0:.2f}".format(x))

else:
    print("ta errado")

File: Java/_1038.java
import java.util.Scanner;
import java.io.IOException;

public class e_1038 
{
 
    public static void main(String[] args) throws IOException 
    {
        Scanner sc = new Scanner(System.in);
        int codigo;
        double total = 0;
        codigo = sc.nextInt();
        double quantidade = sc.nextDouble();

        if (codigo == 1)
        {
            total = quantidade * 4.0;
        }
        else if (codigo == 2)
        {
            total = quantidade * 4.5;
        }
        else if (codigo == 3)
        {
            total = quantidade * 5.0;
        }
        else if (codigo == 4)
        {
            total = quantidade * 2.0;
        }
        else if (codigo == 5)
        {
            total = quantidade * 1.5;
        }
        System.out.printf("Total: R$ %.2f\n", total);
    }
 
}
